### PR TITLE
Gdaval

### PR DESCRIPTION
**Pull request by Daval-G**
This is a great contribution to the current project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ These steps assume that you [have installed git locally](https://www.atlassian.c
 
 2. Clone your forked repo locally. `git clone git@github.com:<your-github-handle>/git-tuto.git`
 
-3. Add the original project as a distant repo. `git remote add https://github.com/zaccharieramzi/git-tuto.git`
+3. Add the original project as a distant repo. `git remote add master https://github.com/zaccharieramzi/git-tuto.git`
```
Comment (zaccharieramzi) [inline README.md]: Actually, the preferred name for the original project distant repo is `upstream`. It is also how I called it in step 10., so I'd rather you change it to that.

But thanks for the change!

File: participants.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
 Zaccharie Ramzi,
-
+Guillaume Daval-Frérot,
```
Comment (zaccharieramzi) [inline participants.csv:2]: Did you add a new line at the end of the file?

**Review by zaccharieramzi — CHANGES_REQUESTED** [review]
Some changes

File: participants.csv (hunk 2 above)
Comment (Daval-G) [reply participants.csv:2]: No, actually I explicitly removed it. I can add it back if it makes sense.

**Review by zaccharieramzi — APPROVED** [review]
LGTM

**Pull request merged**